### PR TITLE
fix: retornando userDTO 

### PR DESCRIPTION
**Pull request by edukatiau**

### CODE REVIEW

**Comment by guilhXavier** [comment]
I think this PR has modifications from another PR?

File: src/main/java/ifsul/agileproject/rachadinha/controller/UserController.java [hunk 1]
```diff
@@ -20,57 +22,58 @@ public UserController(UserService userService) {
...
-            return userService.saveUser(usuario);
-        }else{
-            return new User();
-        }
+    public ResponseEntity updateUser(@RequestBody UserDTO dto, @PathVariable Integer id){
+      if(userService.existsUserByID(id)){
+        User user = userService.findUserByID(id);
+        user.setName(dto.getName());
+        user.setEmail(dto.getEmail());
+        user.setPassword(dto.getPassword());
```
Comment (guilhXavier) [inline src/main/java/ifsul/agileproject/rachadinha/controller/UserController.java:51]: Can you take a look at the Object Mapper pattern for Java? That way, we don't have to build objects by hand like so

Here's [an example](https://github.com/guilhXavier/openfut/tree/main/src/main/java/br/com/gtx/openfut/mapper)

File: src/main/java/ifsul/agileproject/rachadinha/controller/UserController.java [hunk 2]
```diff
@@ -20,57 +22,58 @@ public UserController(UserService userService) {
...
-    @ResponseStatus(HttpStatus.OK)
-    public void deleteUserByID(@PathVariable Integer id){
-        userService.deleteUserByID(id);
+    public ResponseEntity deleteUserByID(@PathVariable Integer id){
+      userService.deleteUserByID(id);
+      return new ResponseEntity("Usuário deletado", HttpStatus.OK);
     }
 
-    //Atualizar
+    //Atualizar usuário pelo ID
```
Comment (guilhXavier) [inline src/main/java/ifsul/agileproject/rachadinha/controller/UserController.java:44]: Please remove these comments before the controller methods

**Review by guilhXavier — CHANGES_REQUESTED** [review]
Take a look at the comments I've left

**Review by JPedro-M — APPROVED** [review]

**Review by guilhXavier — APPROVED** [review]

**Pull request merged**